docs/functions/data.rst

```diff
@@ -99,6 +99,23 @@ Environment
    .. versionchanged:: 2.0
       Return method changed from stdout.
 
+.. function:: _lp_container() -> var:lp_container
+
+   Returns a string matching the container context of the shell. Valid values:
+
+   * ``Sing`` - running in a Singularity container
+   * ``Pdmn`` - running in a Podman container
+   * ``Dokr`` - running in a Docker container
+   * ``LXC`` - running in an LXC container
```

Comment:
I'm not sure about shortening these strings. 6 characters is not much longer than 4, I would rather see the whole thing. I can see how "Singularity" would be a bit long, but maybe just that one is shortened? Or maybe this function returns the whole thing, then `_lp_container_color()` shortens it if it's longer than 8 or so chars (like `_lp_vcs_details_color()` does with a commit ID:
https://github.com/nojhan/liquidprompt/blob/2daf0e453039950d79dcf0eec5a04931e363a92a/liquidprompt#L1436-L1437

Reply:
I'll change to full for all but singularity, which will be "Singlrty" which is only 2 letters longer.